portals/knowledge-management/src/Pages/ViewServiceRequest.vue

```diff
@@ -0,0 +1,312 @@
+                                        </div>
+
+                                        <div class="mb-4">
+                                            <h2 class="text-base font-semibold">Type:</h2>
+                                            <p class="text-gray-700">
+                                                {{ serviceRequest.type_name }}
+                                            </p>
+                                        </div>
+                                    </div>
+                                    <!-- Service Request Updates Section -->
+                                    <div
+                                        class="flex flex-col divide-y ring-1 ring-black/5 shadow-sm px-3 pt-3 pb-1 rounded bg-white mb-4"
+                                    >
+                                        <h2 class="text-xl font-bold mb-4">Service Request Updates</h2>
+
+                                        <div id="updates-list">
+                                            <!-- Updates will be dynamically inserted here -->
+
+                                            <div
+                                                :class="
+                                                    serviceRequestUpdate.internal
+                                                        ? 'mb-4 p-4 bg-gray-50 border border-gray-200 rounded'
+                                                        : 'mb-4 p-4 border border-blue-200 rounded bg-gradient-to-br from-primary-500 to-primary-800 text-white'
+                                                "
+                                                v-for="serviceRequestUpdate in serviceRequestUpdates"
+                                                :key="serviceRequestUpdate.id"
+                                            >
+                                                <p :class="serviceRequestUpdate.internal ? 'text-gray-700' : ''">
+                                                    {{ serviceRequestUpdate.update }}
+                                                </p>
+                                                <span
+                                                    :class="
+                                                        serviceRequestUpdate.internal
+                                                            ? 'text-sm text-gray-500'
+                                                            : 'text-sm'
+                                                    "
+                                                >
+                                                    {{ serviceRequestUpdate.created_at }}
+                                                </span>
+                                            </div>
+
+                                            <!-- More updates can be added here -->
+                                        </div>
+                                    </div>
+
+                                    <!-- Add New Update Form -->
```

Comment:
Let's move this to be above the updates. Similiar to Jira

Reply:
Done